mvpa2/datasets/mri.py

```diff
@@ -270,8 +283,18 @@ def fmri_dataset(samples, targets=None, chunks=None, mask=None,
             ds.fa[fattr] = ds.a.mapper.forward1(value)
 
     # store interesting props in the dataset
-    ds.a['imghdr'] = imghdr
-    ds.a['imgtype'] = imgtype
+    ds.a['imgaffine'] = img.get_affine()
+    ds.a['imgtype'] = img.__class__.__name__
```

Comment:
Getting the image type and affine matrix seems duplicated in the convert_dataset function in the new file tools/strip_nibabel. Can we put this functionality in one place?